script.extendedinfo/resources/lib/TheMovieDB.py

```diff
@@ -341,12 +283,12 @@ def handle_movies(results, local_first=True, sortkey="year"):
                         'Trailer': "%splaytrailer&&id=%s" % (PLUGIN_BASE, movie.get("id")),
                         'genre': " / ".join([i for i in genres if i]),
                         'votes': movie.get('vote_count'),
-                        'year': Utils.get_year(release_date),
+                        'year': utils.get_year(release_date),
                         'rating': round(movie['vote_average'], 1) if movie.get('vote_average') else "",
                         'userrating': movie.get('rating'),
                         'premiered': release_date})
         item.set_properties({'id': movie.get("id"),
-                             'popularity': movie.get('popularity'),
+                             'popularity': round(movie['popularity'], 1) if movie.get('popularity') else "",
```

Comment:
It's fine this way, I filter out all empty dict entries before processing any further.